app/packages/core/src/components/Actions/ActionsRow.tsx

```diff
@@ -179,23 +181,23 @@ const Tag = ({
     useEventHandler(lookerRef.current, "pause", () => setAvailable(true));
 
   const baseTitle = `Tag sample${modal ? "" : "s"} or labels`;
-  const title = readOnly
+  const title = isReadOnly
     ? `Can not ${baseTitle.toLowerCase()} in read-only mode.`
```

Comment:
Thoughts on splitting this message? For example:

- Has permission and is not read-only: Tag samples or labels
- Does not have permission: You do not have permission to tag samples or labels
- Readonly mode: Can not tag samples or labels in read-only mode.

Reply:
Just saw this. Yes, I will implement it in the messages.